src/serial.c

```diff
@@ -12,7 +13,7 @@ static char receive_buffer[RECEIVE_BUFFER_SIZE];
  */
```

Comment:
Updated the header to explain the mutex lock

Reply:
92eadcbafc0207b1ea60ca9e0544cf4c1456c890